test/credential.policies.e2e-spec.ts

```diff
@@ -137,13 +140,13 @@ describe('Issue and Prove credentials using policies (e2e)', () => {
                     throw e;
                 }
             });
-    }, 30000);
+    });
 
     it ('issuer creates credential definition', async () => {
         await delayFunc(1000);
         const data = {
             schema_id: schemaId,
-            support_revocation: true,
+            support_revocation: false,
```

Comment:
Setting revocation to true was giving me errors about now revocation registry present